packages/components/src/font-size-picker/index.js

```diff
@@ -48,7 +48,7 @@ function getSelectOptions( optionsArray, disableCustomFontSizes ) {
 		name: option.name,
 		size: option.size,
 		style: {
-			fontSize: `min( ${ option.size }, ${ MAX_FONT_SIZE_DISPLAY } )`,
+			fontSize: `min( ${ option.size }px, ${ MAX_FONT_SIZE_DISPLAY } )`,
```

Comment:
The `size` is a value defined by the theme. Some themes seem to make this a px value (e.g `12px`) while others just define it as a number (`12`)

It looks like when #26705 was tested it was using a theme that just defined it as a number (Twenty Twenty One). Testing this PR in Twenty Twenty One, and things are broken.

This code will have to check if the `option.size` ends with `'px'`, and only add it if it doesn't.